diff -rupN a/Makefile.in b/Makefile.in
--- a/Makefile.in	2012-07-15 13:18:35.000000000 -0500
+++ b/Makefile.in	2012-12-01 00:25:03.077411784 -0600
@@ -160,17 +160,17 @@ localinstall_opera:
 	make localinstall BROWSERDIR=.opera
 
 install:
-	-@install -d @bindir@
-	-@install -d $(prefix)$(libprefix)/mozilla/plugins
-	-@install -d @mandir@/man7
-	-@install -d @sysconfdir@
-	install mozplugger-helper     @bindir@
-	install mozplugger-controller @bindir@
-	install mozplugger-linker     @bindir@
-	for a in ${PLUGINDIRS}; do install mozplugger.so $$a ; done
-#	cp mozplugger.so $(prefix)$(libprefix)/mozilla/plugins/
-	install $(srcdir)/mozpluggerrc  @sysconfdir@
-	install $(srcdir)/mozplugger.7  @mandir@/man7/
+	-@install -d ${root}@bindir@
+	-@install -d ${root}$(prefix)$(libprefix)/nsbrowser/plugins
+	-@install -d ${root}@mandir@/man7
+	-@install -d ${root}@sysconfdir@
+	install mozplugger-helper     ${root}@bindir@
+	install mozplugger-controller ${root}@bindir@
+	install mozplugger-linker     ${root}@bindir@
+	for a in ${root}$(prefix)$(libprefix)/nsbrowser/plugins; do install mozplugger.so $$a ; done
+#	cp mozplugger.so ${root}$(prefix)$(libprefix)/mozilla/plugins/
+	install $(srcdir)/mozpluggerrc  ${root}@sysconfdir@
+	install $(srcdir)/mozplugger.7  ${root}@mandir@/man7/
 
 uninstall:
 	rm -f @mandir@/man7/mozplugger.7
