--- goldendict.pro.orig	2016-07-28 23:48:03.233297827 +0700
+++ goldendict.pro	2016-07-28 23:49:51.925801495 +0700
@@ -36,7 +36,8 @@
 QT += sql
 CONFIG += exceptions \
     rtti \
-    stl
+    stl \
+    qtsingleapplication
 OBJECTS_DIR = build
 UI_DIR = build
 MOC_DIR = build
@@ -632,5 +633,3 @@
 TS_OUT ~= s/.ts/.qm/g
 PRE_TARGETDEPS += $$TS_OUT
 
-include( qtsingleapplication/src/qtsingleapplication.pri )
-
