--- src/Makefile.bak	2012-04-08 15:07:27.781783292 +0400
+++ src/Makefile	2012-04-08 15:07:53.931781355 +0400
@@ -265,7 +265,7 @@
 	$(MAKE) $(PROJ) \
 		JOHN_OBJS="$(JOHN_OBJS) $(CUDA_OBJS) c3_fmt.o x86-64.o sse-intrinsics.o" \
 		CFLAGS="$(CFLAGS) -DHAVE_CRYPT -DHAVE_DL -DHAVE_CUDA" \
-		LDFLAGS="$(LDFLAGS)  -lcrypt -ldl -lcudart 
+		LDFLAGS="$(LDFLAGS)  -lcrypt -ldl -lcudart -L/opt/cuda/lib64/" 
 
 linux-x86-64:
 	$(LN) x86-64.h arch.h
@@ -363,7 +363,7 @@
 	$(MAKE) $(PROJ) \
 		JOHN_OBJS="$(JOHN_OBJS) $(CUDA_OBJS) c3_fmt.o x86.o x86-sse.o sha1-mmx.o md4-mmx.o md5-mmx.o" \
 		CFLAGS="$(CFLAGS) -DHAVE_CRYPT -DHAVE_DL -DHAVE_CUDA" \
-		LDFLAGS="$(LDFLAGS)  -lcrypt -ldl -lcudart 
+		LDFLAGS="$(LDFLAGS)  -lcrypt -ldl -lcudart -L/opt/cuda/lib64/" 
 
 linux-x86-sse2:
 	$(LN) x86-sse.h arch.h
