--- Makefile.in.org	2012-11-09 20:18:26.123908225 +0100
+++ Makefile.in	2012-11-09 20:18:56.756939600 +0100
@@ -371,9 +371,9 @@
 ifeq ($(OPAL_JAVA), yes)
 
 JAVA_SRCDIR      = $(OPAL_SRCDIR)/java
-JAVA_WRAPPER     = $(JAVA_SRCDIR)/java_swig_wrapper.c
+JAVA_WRAPPER     = $(JAVA_SRCDIR)/java_swig_wrapper.cxx
 
-VPATH_C         += $(JAVA_SRCDIR)
+VPATH_CXX       += $(JAVA_SRCDIR)
 SOURCES         += $(JAVA_WRAPPER)
 
 endif

