--- a/sysdeps/x86_64/multiarch/strstr.c       2013-08-11 02:52:55.000000000 +0400
+++ b/sysdeps/x86_64/multiarch/strstr.c   2013-10-21 21:39:56.699250134 +0400
@@ -82,11 +82,11 @@
    5.  failed string compare, go back to scanning
  */
 
-#if !(defined USE_AS_STRCASESTR && defined STRCASESTR_NONASCII)
 /* Simple replacement of movdqu to address 4KB boundary cross issue.
    If EOS occurs within less than 16B before 4KB boundary, we don't
    cross to next page.  */
-static __m128i
+
+static inline __m128i
 __m128i_strloadu (const unsigned char * p, __m128i zero)
 {
   if (__builtin_expect ((int) ((size_t) p & 0xfff) > 0xff0, 0))
@@ -99,14 +99,14 @@
     }
   return _mm_loadu_si128 ((__m128i *) p);
 }
-#endif
 
 #if defined USE_AS_STRCASESTR && !defined STRCASESTR_NONASCII
 
 /* Similar to __m128i_strloadu.  Convert to lower case for POSIX/C
    locale and other which have single-byte letters only in the ASCII
    range.  */
-static __m128i
+
+static inline __m128i
 __m128i_strloadu_tolower (const unsigned char *p, __m128i zero, __m128i uclow,
                          __m128i uchigh, __m128i lcqword)
 {
