--- a/src/py_client.py   2013-11-15 05:16:40.077480336 +0400
+++ b/src/py_client.py       2013-11-15 05:17:26.018045663 +0400
@@ -19,7 +19,6 @@
                    'INT8':   'b', 'int8_t':  'b',
                    'INT16':  'h', 'int16_t': 'h',
                    'INT32':  'i', 'int32_t': 'i',
+                   'INT64':  'i', 'int64_t': 'i',
                    'BYTE': 'B',
                    'BOOL': 'B',
                    'char': 'b',
