diff -urN a/layout/style/nsCSSRuleProcessor.cpp b/layout/style/nsCSSRuleProcessor.cpp
--- a/layout/style/nsCSSRuleProcessor.cpp	2011-03-18 19:34:00.000000000 -0400
+++ b/layout/style/nsCSSRuleProcessor.cpp	2011-04-03 08:12:35.205741696 -0400
@@ -1317,7 +1317,7 @@
     return PR_FALSE;
 
   const nsDefaultStringComparator defaultComparator;
-  const nsASCIICaseInsensitiveStringComparator ciComparator;
+  const nsASCIICaseInsensitiveStringComparator ciComparator = nsASCIICaseInsensitiveStringComparator();
   const nsStringComparator& comparator =
       (aAttrSelector->mCaseSensitive || !isHTML)
                 ? static_cast<const nsStringComparator&>(defaultComparator)
diff -urN a/toolkit/xre/nsAppRunner.cpp b/toolkit/xre/nsAppRunner.cpp
--- a/toolkit/xre/nsAppRunner.cpp	2011-04-03 08:17:35.091741681 -0400
+++ b/toolkit/xre/nsAppRunner.cpp	2011-04-03 08:12:12.305741698 -0400
@@ -1003,7 +1003,7 @@
 
 #endif
 
-static const nsXULAppInfo kAppInfo;
+static const nsXULAppInfo kAppInfo = nsXULAppInfo();
 static nsresult AppInfoConstructor(nsISupports* aOuter,
                                    REFNSIID aIID, void **aResult)
 {
diff -urN a/xpcom/base/nsDebugImpl.cpp b/xpcom/base/nsDebugImpl.cpp
--- a/xpcom/base/nsDebugImpl.cpp	2011-03-18 19:34:13.000000000 -0400
+++ b/xpcom/base/nsDebugImpl.cpp	2011-04-03 08:12:15.530741697 -0400
@@ -511,7 +511,7 @@
 #endif
 }
 
-static const nsDebugImpl kImpl;
+static const nsDebugImpl kImpl = nsDebugImpl();
 
 nsresult
 nsDebugImpl::Create(nsISupports* outer, const nsIID& aIID, void* *aInstancePtr)
diff -urN a/xpcom/base/nsTraceRefcntImpl.cpp b/xpcom/base/nsTraceRefcntImpl.cpp
--- a/xpcom/base/nsTraceRefcntImpl.cpp	2011-03-18 19:34:13.000000000 -0400
+++ b/xpcom/base/nsTraceRefcntImpl.cpp	2011-04-03 08:12:15.529741697 -0400
@@ -1356,7 +1356,7 @@
   return NS_OK;
 }
 
-static const nsTraceRefcntImpl kTraceRefcntImpl;
+static const nsTraceRefcntImpl kTraceRefcntImpl = nsTraceRefcntImpl();
 
 NS_METHOD
 nsTraceRefcntImpl::Create(nsISupports* outer, const nsIID& aIID, void* *aInstancePtr)
diff -urN a/xpcom/glue/nsEnumeratorUtils.cpp b/xpcom/glue/nsEnumeratorUtils.cpp
--- a/xpcom/glue/nsEnumeratorUtils.cpp	2011-03-18 19:34:13.000000000 -0400
+++ b/xpcom/glue/nsEnumeratorUtils.cpp	2011-04-03 08:12:15.574741697 -0400
@@ -113,7 +113,7 @@
     return NS_ERROR_UNEXPECTED;
 }
 
-const EmptyEnumeratorImpl EmptyEnumeratorImpl::kInstance;
+const EmptyEnumeratorImpl EmptyEnumeratorImpl::kInstance = EmptyEnumeratorImpl();
 
 nsresult
 NS_NewEmptyEnumerator(nsISimpleEnumerator** aResult)
diff -urN a/xpcom/io/nsUnicharInputStream.cpp b/xpcom/io/nsUnicharInputStream.cpp
--- a/xpcom/io/nsUnicharInputStream.cpp	2011-03-18 19:34:13.000000000 -0400
+++ b/xpcom/io/nsUnicharInputStream.cpp	2011-04-03 08:12:16.023741697 -0400
@@ -464,4 +464,4 @@
 }
 
 const nsSimpleUnicharStreamFactory
-nsSimpleUnicharStreamFactory::kInstance;
+nsSimpleUnicharStreamFactory::kInstance = nsSimpleUnicharStreamFactory();
